src/jaleo/record.rs

```diff
@@ -279,7 +279,7 @@ impl Record {
         let aead = Aes256Gcm::new(key);
 
         let mut iv = [0_u8; AES_IV_LENGTH];
-        thread_rng().fill(&mut iv);
+        //thread_rng().fill(&mut iv);
```

Comment:
This was the source of randomness when minting records. Not sure what the best way would way would be to solve it (guessing it should be based on the RNG used to create the randomizer).